packages/cdk/overlay/src/useOverlay.ts

```diff
@@ -0,0 +1,195 @@
+import type { Instance as PopperInstance } from '@popperjs/core'
+import type { ComputedRef } from 'vue'
+
+import type {
+  OverlayInstance,
+  OverlayOptions,
+  OverlayPopperEvents,
+  OverlayTrigger,
+  OverlayTriggerEvents,
+  RefElement,
+  TriggerElement,
+} from './types'
+
+import { ComponentPublicInstance, computed, reactive, ref, watch } from 'vue'
+import { createPopper } from '@popperjs/core'
+import { toRawType } from '@vue/shared'
+import { uniqueId } from '@idux/cdk/utils'
+
+import { usePopperOptions } from './usePopperOptions'
+
+const isHTMLElement = (val: unknown) => toRawType(val).startsWith('HTML')
```

Comment:
```suggestion
const isHTMLElement = (val: unknown): val is HTMLElement => toRawType(val).startsWith('HTML')
```

这个丢到 utils/typeof 里去吧。